content/cpp/concepts/arrays/arrays.md

```diff
@@ -59,3 +59,13 @@ std::cout << vowels[0];
 In the case above, an array of `chars` was initialized with all the vowels, and then the first element in the array at index `0` was printed out. Then the element at index `0` was modified by being assigned a new value of `'r'`, which was then printed out.
 
 Arrays in C++ have a fixed size, meaning elements can not be added or removed once the array has been created. Only existing elements may be modified without changing the total size or shape of the structure.
+
+## Uninitialized Elements
+
+Array elements that are not assigned a value when the array is created are known as uninitialized, and should not be accessed during runtime. For example:
```

Comment:
Array elements that are not assigned a value when the array is created are known as uninitialized and should not be accessed during runtime. For example:"

I removed the comma after "uninitialized" to streamline the sentence structure.